### ISSUE TITLE
Mistake on type declaration

### ISSUE THREAD

**Issue by TsXor** [ktnrg45/vs-code-cython#2]
```
cdef cnp.ndarray[double, ndim=3] fimg
```
which is legal in cython, raises:
```
cdef cnp.ndarray[double, ndim=3] fimg
                              ^
Axis specification only allowed in the "step" slot for a view
```

**Comment by TsXor** [comment]
Another problem: declarations in `pxd` file.
Solution: when checking the grammar of a `pyx` file, if there is a `pxd` file with the same name in the same folder, concat that `pxd` file before the `pyx` file.

**Comment by TsXor** [comment]
Type casting from pointer to typed memoryview:
```
<num_t[:h, :w]>
```
will raise:
```
<num_t[:h, :w]>
      ^
Expected cast close

<num_t[:h, :w]>
       ^
Expected expression
```

**Comment by TsXor** [comment]
declaration of a `ctuple` like
```
cdef (int, double) xy = [x, y]
```
is not supported

**Comment by ktnrg45** [comment]
Thanks for reporting these issues.

pxd files should be imported automatically and ctuple parsing has been added in version 0.1.0.

I will take a look at the other issues when I have time.

**Comment by ktnrg45** [comment]
All of these issues should be resolved in 0.1.2
